codechecker_lib/analyzers/analyzer_types.py

```diff
@@ -330,52 +317,37 @@ def construct_result_handler(args,
     """
     Construct a result handler.
     """
+    assert buildaction.analyzer_type in supported_analyzers, \
```

Comment:
See my answer above. Right now the other parts of the code terminates when one of the analyzers is unsupported.